lib/msal-core/README.md

```diff
@@ -49,6 +49,9 @@ We offer two methods of storage for Msal, `localStorage` and `sessionStorage`.
 ### forceRefresh to skip cache
 If you would like to skip a cached token and go to the server, please pass in the boolean `forceRefresh` into the `AuthenticationParameters` object used to make a login / token request. **WARNING:** This should not be used by default, because of the performance impact on your application.  Relying on the cache will give your users a better experience, and skipping it should only be used in scenarios where you know the current cached data does not have up to date information.  Example: Admin tool to add roles to a user that needs to get a new token with updates roles.
 
+### Conditional Access and Claims Requests
+MSAL.js supports [conditional access scenarios](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-handling-exceptions#javascript) enabled in Azure AD. You may also [request additional claims]() for your application. Please review the linked MSAL Wiki pages. To understand more about how to enable conditional access for your application, or common conditional access scenarios, please review the docs [here](https://docs.microsoft.com/en-us/azure/active-directory/conditional-access/overview).
```

Comment:
Looks like you forgot the url here: `[request additional claims]()`

Reply:
Yes waiting until the page is up.